server/src/main/java/org/jetbrains/bsp/bazel/server/sync/languages/java/JavaLanguagePlugin.java

```diff
@@ -105,6 +113,37 @@ protected void applyModuleData(JavaModule javaModule, BuildTarget buildTarget) {
     buildTarget.setData(jvmBuildTarget);
   }
 
+  @Override
+  public Path getSourceSet(Path source) {
+    String sourcePackage = this.findPackage(source);
+    if (sourcePackage==null) {
+      return null;
+    }
+    Path source_package_path = Paths.get(sourcePackage.replace(".", "/"));
+
+    return Paths.get("/")
```

Comment:
in kotlin u could do in with simple subtraction

Reply:
In Java when subtracting from absolute path you get relative that's why this is here